sdk/communication/AzureCommunicationChat/Source/Models/Participant.swift

```diff
@@ -28,6 +28,8 @@ import AzureCommunication
 import AzureCore
 import Foundation
 
+// swiftlint:disable force_cast
```

Comment:
Why did we add this swift lint disable here?

Reply:
I force cast the identifier to CommunicationUserIdentifier. This is under the assumption it should always be CommunicationUserIdentifier but I could change it to something like below instead, thoughts?
```
if let user = identifier as? CommunicationUserIdentifier {
        self.user = user
} else {
        throw AzureError.client("Identifier is not a CommunicationUserIdentifier")
}
```

Reply:
Updated to remove force_cast